data/src/main/java/com/globant/data/database/HarryPotterRoomDao.kt

```diff
@@ -13,4 +14,10 @@ interface HarryPotterRoomDao {
 
     @Insert(onConflict = OnConflictStrategy.IGNORE)
     fun insertSpell(spell: SpellRoom)
+
+    @Query("SELECT * FROM house_table WHERE name = :name")
+    fun getHouseByName(name: String): List<HouseRoom>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
be carefull with this strategy, on the other hand, why do you need a strategy here?

Reply:
I need this strategy to say what to do if the house that is trying to insert already exist in the table. But I will refactor to REPLACE instead of IGNORE; so the table will be update with the lastest version of each house. Thanks Mau!